app/src/main/java/com/android/unio/model/event/EventCard.kt

```diff
@@ -72,22 +73,24 @@ fun EventCard(event: Event, onClick: () -> Unit) {
                   color = Color.Black)
 
               Spacer(modifier = Modifier.width(6.dp))
-
+              val type: EventType =
+                  if (event.types.isEmpty()) {
+                    EventType.OTHER
+                  } else event.types[0]
               Box(
                   modifier =
                       Modifier.clip(RoundedCornerShape(4.dp))
-                          .background(addAlphaToColor(event.types.get(0).color, 200))
+                          .background(addAlphaToColor(type.color, 200))
                           .wrapContentWidth()) {
                     Text(
-                        text = event.types.get(0).text,
+                        text = type.text,
```

Comment:
Nice fix ! We should consider now removing the default value in "Event.kt" to "EventType" as we handle now the case where no type is given.